run3/flow/ML/ComputeV2vsFDFrac.py

```diff
@@ -0,0 +1,323 @@
+"""
+python script for the computation of the prompt or non-prompt v2 via extrapolation
+run: python ComputeV2vsFDFrac.py cfgFileName.yml outFileName.root
+
+"""
+
+import sys
+import argparse
+import uproot
+import os
+import numpy as np
+import yaml
+from ROOT import (
+    TFile,
+    TH1F,
+    TH2F,
+    TCanvas,
+    TLegend,
+    TGraphAsymmErrors,
+    TLatex,
+    gRandom,
+    TGraphErrors,
+    TF1,
+    TH1D,
+    TVirtualFitter,
+    Double_t,
+)  # pylint: disable=import-error,no-name-in-module
+from ROOT import (
+    kBlack,
+    kRed,
+    kAzure,
+    kGreen,
+    kRainBow,
+)  # pylint: disable=import-error,no-name-in-module
+from ROOT import (
+    kFullCircle,
+    kFullSquare,
+    kOpenSquare,
+    kOpenCircle,
+    kOpenCross,
+    kOpenDiamond,
+)  # pylint: disable=import-error,no-name-in-module
+
+sys.path.append('/home/wuct/ALICE/local/DmesonAnalysis')
```

Comment:
we should generalize this line for other analysers

Reply:
Done